src/mono/System.Private.CoreLib/src/System/Runtime/DependentHandle.cs

```diff
@@ -0,0 +1,168 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.CompilerServices
+{
+    internal struct Ephemeron
+    {
+        public object? key;
+        public object? value;
```

Comment:
Does the mono runtime care about these exact names?  If not, Key and Value.

Reply:
I don't think it does, as far as I can tell 🤔
The `Ephemeron` type is defined here:

https://github.com/dotnet/runtime/blob/155543fbfcb3c8beda28b8400444b9c34e4d4ca2/src/mono/mono/metadata/sgen-mono.c#L561-L564

So I think as long as the layout matches, the field names on the C# side won't matter.
I'll change this and the field name as well like you mentioned in the comment above 👍